fast/stages/2-networking-c-nva/nva.tf

```diff
@@ -75,13 +75,29 @@ module "nva-template" {
       nat       = false
       addresses = null
     },
+    {
+      network = module.dmz-vpc.self_link
```

Comment:
local.routing_config and var.gcp_ranges need to be updated so I suspect routing is somehow broken (same applies to the other NICs below). 
Please make sure you create test vms (in test-resources.tf) to test the different communication flows.